src/app/containers/FrontPageSection/index.jsx

```diff
@@ -84,6 +112,36 @@ StoryPromoComponent.propTypes = {
   storyNumber: number.isRequired,
 };
 
+const UsefulLinksComponent = ({ items, script, service }) => {
```

Comment:
I think it would be nice if `UsefulLinksComponent` was in it's own file though, that way we could Isolate it to have it's own stories and test the loogic